pkg/dispatcher/dispatcher.go

```diff
@@ -217,6 +223,128 @@ func (d *Dispatcher) dispatch(ctx context.Context, msg amqp.Delivery, ceClient c
 	}
 }
 
+func (d *Dispatcher) dispatchDLQ(ctx context.Context, msg amqp.Delivery, ceClient cloudevents.Client, channel rabbit.RabbitMQChannelInterface) {
+	start := time.Now()
+	if _, ok := msg.Headers["knativeerrordest"]; ok {
```

Comment:
In the case of the dispatch DLQ method, remember that is the code we had before your changes, so this function should be the other code